src/main/resources/static/styles.css

```diff
@@ -0,0 +1,101 @@
+* {
+    padding: 0;
+    margin: 0;
+}
+
+.wrapper {
+    width: 100%;
+}
+
+#titleImg {
+ width: 400px;
+
```

Comment:
Can we clean up some of this white space between the attribute and the closing bracket?